doc/decisions/0005-application-errors-when-the-academy-api-errors.md

```diff
@@ -0,0 +1,58 @@
+# 5. Application errors when the Academy API errors
+
+Date: 2022-07-15
+
+## Status
+
+Accepted
+
+## Context
+
+Right now the application is wholly reliant on the 'Academies API' aka 'TRAMS
+Data API' aka 'Prepare API' [1] to render almost all requests. Whilst the
+application could try to render the data it can retrieve, there would be little
```

Comment:
Maybe this needs something like "in the case of a failure the application could try to render..."